zamba/models/config.py

```diff
@@ -603,7 +603,10 @@ def preprocess_labels(cls, values):
         labels = (
             pd.get_dummies(labels.rename(columns={"label": "species"}), columns=["species"])
             .groupby("filepath")
-            .max()
+            # instead of collapsing multiple rows with the same video into one row, keep them as is
```

Comment:
In the edge case @pjbull gave, v1.mp4 has two labels, antelope and cow. This means there are both antelope and cow in the video. 
The splits.csv before running the changed code:
filepath,split
v0.mp4,train
v1.mp4,train
v2.mp4,holdout
v4.mp4,val
v5.mp4,holdout
The splits.csv after running the changed code:
v0.mp4,train
v1.mp4,val
v1.mp4,train
v2.mp4,holdout
v4.mp4,val
v5.mp4,holdout

v1.mp4 is assigned twice because it includes 2 species. If a given video is only assigned to one set, the splits before changing the code is OK; why would we need to change anything?